### DISCUSSION TITLE
The maximum length should be in terms of token count rather than character count.

### DISCUSSION THREAD

**Discussion by secsilm** [langgenius/dify#3592] [Suggestion]
LLM is billed by the number of tokens, not by the number of characters, so the maximum length in characters is meaningless.

![image](https://github.com/langgenius/dify/assets/8894643/55397f60-21bf-410c-87c8-c8410a281133)

**Comment by BradKML** [comment]
You do realize that tokenization changed between different LLMs, right? You can only approximate based on things like "100 tokens are 75 words" and add margins on top.
